_opensearch/query-dsl/term.md

```diff
@@ -2,26 +2,40 @@
 layout: default
 title: Term-level queries
 parent: Query DSL
-nav_order: 30
+nav_order: 70
 ---
 
 # Term-level queries
 
-OpenSearch supports two types of queries when you search for data: term-level queries and full-text queries.
+Term-level queries search an index for all documents that contain a search term. Documents returned by a term-level query are not sorted by their relevance scores.
 
-The following table describes the differences between them:
+When working with text data, use term-level queries for fields mapped as keyword only.
 
-| | Term-level queries | Full-text queries
+Term-level queries are not suited for searching analyzed text fields. To return analyzed fields, use a full-text query.
+
+## Term-level query types
+
+You can use any of the term-level query types shown in the following table:
+
+| Query type | Description
 :--- | :--- | :---
-*Description* | Term-level queries answer which documents match a query. | Full-text queries answer how well the documents match a query.
-*Analyzer* | The search term isn't analyzed. This means that the term query searches for your search term as it is.  | The search term is analyzed by the same analyzer that was used for the specific field of the document at the time it was indexed. This means that your search term goes through the same analysis process that the document's field did.
-*Relevance* | Term-level queries simply return documents that match without sorting them based on the relevance score. They still calculate the relevance score, but this score is the same for all the documents that are returned. | Full-text queries calculate a relevance score for each match and sort the results by decreasing order of relevance.
-*Use Case* | Use term-level queries when you want to match exact values such as numbers, dates, tags, and so on, and don't need the matches to be sorted by relevance. | Use full-text queries to match text fields and sort by relevance after taking into account factors like casing and stemming variants.
+`term` | Specifies a search for an exact term in the field provided.
+`terms` | Specifies a search for multiple terms in the same provided field.
+`ids` | Specifies a search for one or more documents by the ID value.
+`range` | Specifies a search within an upper and lower bound of values in a provided field.
+`prefix` | Specifies a search for terms that begin with a prefix.
+`exists` | Specifies a search for documents that contain a specific field.
+`wildcard` | Specifies a search for terms that match a wildcard pattern. Indicate matching zero or more characters with an asterisk (*) or a single character by a question mark (?).
+`regexp` | Specifies a search for terms that match a regular expression.
+`terms_set` | Similar to a `terms` query, but you specify the number of matching terms that are required to return a document.
```

Comment:
Please add the `fuzzy` query in this list. It is a subtype of term-level queries.

Reply:
@alicejw-aws , @macohen is taking another look at this from a technical standpoint since we want to make sure it is 100% technically correct before publishing it.

Reply:
@hdhalter thanks, I appreciate that.